openshift/config/common/vars.yaml

```diff
@@ -1,6 +1,6 @@
 # Component Versions
 agnosticv_operator_version: v0.0.24
```

Comment:
I guess we need  a new version of agnosticv-operator.

we should uniform the version scheme, and create the new agnosticv-operator version as `0.0.25`

Reply:
Looking around... it seems just about everyone uses `vX.Y.Z` like you used with agnosticv-operator. I'm thinking it is probably anarchy, poolboy, and k8s_config that should change to match.

Reply:
Ah! I see @tonykay went without the `v` with ftl!

Reply:
What was decided in the end ?